modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -1240,13 +1002,14 @@ static void run_absdiffc(Buffer &dst, const View &src, const float scalar[])
 
     int width = dst.length();
     int chan = dst.meta().chan;
+    const int length = width * chan;
 
     int w = 0;
 #if CV_SIMD
-    w = absdiffc_simd(in, scalar, out, width, chan);
+    w = absdiffc_simd(in, scalar, out, length, chan);
```

Comment:
it was a bug before?

Reply:
No, it wasn't a bug. I just took the length calculation to a higher level.

Reply:
hm, it was `width` then became `width * chan` here - so data amount as increased in `chan` times. Looks like we didn't use by whole data range before (was bug) or processing more data (possible out-of-bound ) for now

Reply:
No, it wasn't so. Earlier we calculate length inside absdiffc_simd_c3() and absdiffc_simd_c1c2c4() functions and then used length for a loop. You can see it [here](https://github.com/opencv/opencv/pull/21204/files#diff-fe0ac6b5a07c1bec59c3756100eb186c9117a82dd1d951ef6997b8423a89943dL1202) for example. Now length is calculated 2 level up in call stack (i.e. here).